ensembl_metadata/models/assembly.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+
+class Assembly(models.Model):
+    assembly_id = models.AutoField(primary_key=True)
+    assembly_accession = models.CharField(unique=True, max_length=16, blank=True, null=True)
```

Comment:
Can the accession be null?

Reply:
In the schema here, yes - we might have species without an accession, even though that's not supposed to happen.

In the pending schema with datasets, however, we can shift some data to the 'assembly' dataset, and make the accession unique and mandatory in the model (https://github.com/Ensembl/ensembl-metadata-registry/blob/feature/schema_update/ensembl_metadata/models/assembly.py). Which makes things a bit more intuitive, I think.

Reply:
Me too, and I've done that in the feature/schema_update branch - but without the dataset table, that can't be enforced.